msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -399,7 +399,20 @@ async def on_teams_channel_restored(
 
 ### Team members added
 
-The `teamMemberAdded` event is sent to your bot the first time it is added to a conversation. The event is sent to your bot every time a new user is added to a team or group chat where your bot is installed. The user information that is ID, is unique for your bot and can be cached for future use by your service, such as sending a message to a specific user.
+The `teamMemberAdded` event is sent to your bot in the following scenarios:
```

Comment:
@vule-msft do we set eventType=teamMemberAdded even when this conversationUpdate is for 1:1 or group chat?
<img width="268" alt="image" src="https://user-images.githubusercontent.com/767208/170321374-1655daee-c712-4c97-a056-afb3c32cf927.png">

Reply:
I think we would need to make similar updates for the team member removed section

Reply:
I like that.